examples/image-builder.yaml

```diff
@@ -23,6 +23,7 @@ vars:
   new_image_family: my-slurm-image
   network_name: image-builder
   subnetwork_name: image-builder-us-central1
+  disk_size: 100
```

Comment:
I would try to set it to something smaller. 100GB is pretty large here. 
I believe the schedmd Image is 32GB, so anything larger than that is ok.

Reply:
Reduced to 32GB